packages/core/src/utils.ts

```
@@ -1,11 +1,17 @@
+import vm from 'node:vm'
+import path from 'node:path'
+import { createRequire } from 'node:module'
 import Debug from 'debug'
-import { type ModuleNode, type ViteDevServer, normalizePath } from 'vite'
+import type { ModuleNode, ViteDevServer } from 'vite'
+import { isCallOf } from 'ast-kit'
```

ESLint

[error] 'isCallOf' is defined but never used.

(unused-imports/no-unused-imports)